Jenkinsfile

```diff
@@ -13,16 +13,18 @@ ansiColor('gnome-terminal') {
           sh """sudo -E ci/pipeline jenkins"""
         }
       } finally {
-        junit(allowEmptyResults: true, testResults: 'target/test-reports/*.xml')
-        junit allowEmptyResults: true, testResults: 'target/test-reports/*integration/*.xml'
+        //junit(allowEmptyResults: true, testResults: 'target/test-reports/*.xml')
```

Comment:
is this temporary?

Reply:
Of course :). This is for faster iterations on Jenkins.